base/tomcat-10.1/.classpath

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file is not needed. There's already an Eclipse project in the top-level folder.